### PR TITLE
Docker compose example

### PR DESCRIPTION
**Pull request by yaskoo**
cc @tokovach

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM maven:3-openjdk-11 AS builder
```
Comment (nikolaykolev) [inline Dockerfile:1]: there's a slim version, although not much smaller

File: README.md [hunk 2]
```diff
@@ -1,2 +1,31 @@
 # kafka-sink-graphdb
-Kafka Sink Connector for Smart Update streaming to GraphDB 
+Kafka Sink Connector for Smart Update streaming to GraphDB
+
+# Docker & Docker Compose
+
+A [Dockerfile](./Dockerfile) is available for building the sink connector as a docker image.
+It is a multistage dockerfile which builds and tests the connector and in the final stage copies the connector on `plugin.path`.
```
Comment (nikolaykolev) [inline README.md:7]: double 'and'

File: README.md [hunk 3]
```diff
@@ -1,2 +1,31 @@
...
+The image is based on confluent kafka connect [confluentinc/cp-kafka-connect](https://hub.docker.com/r/confluentinc/cp-kafka-connect) image.
+
+Inside the [docker-compose](./docker-compose) directory there is an example compose file which sets everything up - zookeeper, kafka, graphdb and kafka connect.
+In the same directory the [run.sh](./docker-compose/run.sh) script can be used to quickly test the sink connector.
+
+The script will do the following:
+
+1. execute `docker-compose up -d` to start the required services
+    * if the connector image is not present locally it will be build
+1. wait for graphdb and kafka connect to start
```
Comment (nikolaykolev) [inline README.md:18]: it's good to write graphdb as GraphDB

File: docker-compose/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+      KAFKA_METRIC_REPORTERS: io.confluent.metrics.reporter.ConfluentMetricsReporter
+      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
+      KAFKA_GROUP_INITIAL_REBALANCE_DELAY_MS: 0
+      CONFLUENT_METRICS_REPORTER_BOOTSTRAP_SERVERS: broker:29092
+      CONFLUENT_METRICS_REPORTER_ZOOKEEPER_CONNECT: zookeeper:2181
+      CONFLUENT_METRICS_REPORTER_TOPIC_REPLICAS: 1
+      CONFLUENT_METRICS_ENABLE: 'true'
+      CONFLUENT_SUPPORT_CUSTOMER_ID: anonymous
+
+  graphdb:
```
Comment (nikolaykolev) [inline docker-compose/docker-compose.yml:37]: it's good to put some additional jvm args such as xmx

File: pom.xml [hunk 5]
```diff
@@ -101,114 +103,114 @@
...
+			<version>3.7.2</version>
+		</dependency>
+		<dependency>
+			<groupId>commons-logging</groupId>
+			<artifactId>commons-logging</artifactId>
+			<version>1.1.1</version>
+		</dependency>
+		<dependency>
+			<groupId>org.eclipse.rdf4j</groupId>
+			<artifactId>rdf4j-rio-nquads</artifactId>
```
Comment (nikolaykolev) [inline pom.xml:160]: not sure if nquads is enough, what about other formats. If other formats can be used, but not imported, rdf4j will throw some nasty npe's

Comment (yaskoo) [reply pom.xml:160]: @tokovach I think somebody mentioned that the other format are work in progress?

File: docker-compose/docker-compose.yml (hunk 4 above)
Comment (yaskoo) [reply docker-compose/docker-compose.yml:37]: I think that for demo purposes this is fine

File: pom.xml (hunk 5 above)
Comment (tokovach) [reply pom.xml:160]: Yasko please look at all the file formats added in GraphDB's POM file and add them here as well.

**Review by nikolaykolev — APPROVED** [review]

File: pom.xml (hunk 5 above)
Comment (yaskoo) [reply pom.xml:160]: will they be used?

**Pull request merged**